src/app/home/home.component.ts

```diff
@@ -76,11 +76,7 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
         this.http.get(this.pingURL).subscribe({
             next: () => console.log('pinging server...'),
             error: (e: HttpErrorResponse) => {
-                if (e.status === 429) {
-                    this.disableEditor('Tepër kërkesa për shenjime për momentin')
-                } else {
-                    this.disableEditor('Fatkeqësisht kemi një problem me serverat. Ju kërkojmë ndjesë, ndërsa kërkojme për një zgjidhje.' );
-                }
+               this.disableEditor(e);
```

Comment:
One more minor thing, can you remove {, }, and ;? Also, run prettier.

Approved.